packages/web/src/scss/components/Modal/_Modal.scss

```diff
@@ -21,18 +21,10 @@
     overscroll-behavior-y: contain;
 }
 
-.Modal__backdrop {
-    position: fixed;
-    z-index: -1;
+.Modal::backdrop {
     background-color: theme.$backdrop-background-color;
-    visibility: hidden;
-    opacity: 0;
-    inset: 0;
-
-    @media (prefers-reduced-motion: no-preference) {
-        transition-property: opacity, visibility;
-        transition-duration: 0.25s;
-    }
+    visibility: visible;
```

Comment:
do we actually need these (visibility and opacity)?

Reply:
Yes, since there is a transition.

https://stackoverflow.com/a/27900094